src/Core/Prefab/Planar/PlanarTileBuilder.js

```diff
@@ -1,73 +1,72 @@
+/* eslint-disable class-methods-use-this */
```

Comment:
This may be outside the scope of this PR, but I think we should drop this rule: I don't see the value in having always `this` used in class methods.

Reply:
rule dropped